metered/src/metric.rs

```diff
@@ -60,16 +60,40 @@ impl<'a, R, M: Metric<R>> Drop for ExitGuard<'a, R, M> {
 /// A trait for Counters
 pub trait Counter: Default + Clear + Clearable + Serialize {
     /// Increment the counter
-    fn incr(&self);
+    fn incr(&self) {
+        self.incr_by(1)
+    }
```

Comment:
That's a good point, it would still be nice to provide `incr` over all implementors of the trait but it would require a sealed trait extension that would have to be brought into scope, I'm not sure this is worth the effort bulletproofing for.